assets/src/settings-page/toggle-use-native-img-tag.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * WordPress dependencies
+ */
+import { createInterpolateElement, useContext } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { AMPSettingToggle } from '../components/amp-setting-toggle';
+import { Options } from '../components/options-context-provider';
+import { Loading } from '../components/loading';
+
+/**
+ * Use native img tag instead of amp-img tag toggle on the settings page.
+ */
+export function ToggleUseNativeImgTag() {
+	const { editedOptions, fetchingOptions, updateOptions } = useContext( Options );
+
+	if ( fetchingOptions ) {
+		return <Loading />;
+	}
+
+	const useNativeImgTag = editedOptions?.use_native_img_tag;
+
+	return (
+		<section className="use-native-img-tag">
+			<AMPSettingToggle
+				checked={ true === useNativeImgTag }
+				title={ __( 'Use native img tag', 'amp' ) }
```

Comment:
Maybe this would make it more clear for non-technical users:

```suggestion
				title={ __( 'Use native HTML image tag', 'amp' ) }
```